src/content/components/Router/Router.js

```diff
@@ -27,6 +27,11 @@ const RouterNav = withRouter(class _RouterNav extends React.PureComponent {
       {route.label}
     </NavLink></li>);
   }
+
+  refreshMetas() {
+    fetch('/refresh_metas').then(location.reload);
```

Comment:
I would do this:

```js
async refreshMetas() {
  await fetch("/refresh_metas");
  window.location.reload();
}

just to prevent the `response` from fetch being passed to `reload()`. (`reload(true)` actually causes the page to be force-reloaded, which is not really necessary here)
```

Reply:
Yes that's a good point, thank you!